runtime/src/accounts.rs

```diff
@@ -2089,69 +2104,79 @@ mod tests {
 
     #[test]
     fn test_load_accounts_multiple_loaders() {
-        let mut accounts: Vec<TransactionAccount> = Vec::new();
-        let mut error_counters = TransactionErrorMetrics::default();
-
-        let keypair = Keypair::new();
-        let key0 = keypair.pubkey();
-        let key1 = Pubkey::new(&[5u8; 32]);
-        let key2 = Pubkey::new(&[6u8; 32]);
-
-        let mut account = AccountSharedData::new(1, 0, &Pubkey::default());
-        account.set_rent_epoch(1);
-        accounts.push((key0, account));
-
-        let mut account = AccountSharedData::new(40, 1, &Pubkey::default());
-        account.set_executable(true);
-        account.set_rent_epoch(1);
-        account.set_owner(native_loader::id());
-        accounts.push((key1, account));
-
-        let mut account = AccountSharedData::new(41, 1, &Pubkey::default());
-        account.set_executable(true);
-        account.set_rent_epoch(1);
-        account.set_owner(key1);
-        accounts.push((key2, account));
-
-        let instructions = vec![
-            CompiledInstruction::new(1, &(), vec![0]),
-            CompiledInstruction::new(2, &(), vec![0]),
-        ];
-        let tx = Transaction::new_with_compiled_instructions(
-            &[&keypair],
-            &[],
-            Hash::default(),
-            vec![key1, key2],
-            instructions,
-        );
+        let initial_rent_epoch = 1;
+        for (_expected_rent_epoch, set_exempt_rent_epoch_max) in
```

Comment:
we don't use `_expected_rent_epoch`?

Reply:
removed this variation from tests that were unaffected.